Doc/reference/datamodel.rst

```diff
@@ -1000,6 +1001,10 @@ Special attributes:
       A tuple containing the :ref:`type parameters <type-params>` of
       a :ref:`generic class <generic-classes>`.
 
+   :attr:`__static_attributes__`
+      A tuple containing names of attributes of this class which are accessed
+      through ``self.X`` from any function in its body.
+
```

Comment:
The boss said I can ignore this warning: 
<img width="774" alt="image" src="https://github.com/python/cpython/assets/1055913/c4984435-fd30-4c74-a985-baececf8bc85">